README.md

```diff
@@ -75,6 +75,11 @@ ontology to validate against.  Beware that validating large KBs can take a long
 a sample of a validator param file in `sample_params/validate.common_corpus.single.params`
 * To run the validator using the Kotlin version, run `target/appassembler/bin/validateAIF`.
 
+# Running the Ontology Resource Generator
+
+To generate the resource variables from a particular ontology file, please refer to 
+the README located at `src/test/java/com/ncc/aif/ont2javagen/README.md`.
+
```

Comment:
The file is in src/main/java, not src/test/java.  But I actually kind of prefer it in src/test/java, since using this generator on a new ontology will generally involve writing a new (or seriously updating) ExamplesAndValidationTests.java.  But I'm okay with either.

Reply:
It should be in src/main/java.  It is not a unit test.  Also, to Darren's comment, the ExamplesAndValidationTests.java is not going to be the only use of this (although it might be OUR only use of it).